storybook-utilities/components/AdditionalInputInfo.js

```diff
@@ -0,0 +1,162 @@
+          additionalClasses={additionalListItemClasses}
+        >
+          Each input element&apos;s id should be unique on the page.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          The
+          <code> type </code>
+          attribute for an Input should be the most
+          appropriate value that semantically matches your use case.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          The
+          <code> data-id </code>
+          property is provided as a hook for automated tools.
+          If you have multiple instances of the same variant
+          of a component on the same page, make sure each instance
+          has a unique
+          <code> data-id </code>
+          property (&quot;input-text-1&quot;,
+          &quot;input-text-2&quot;, &quot;input-date-1&quot;, etc).
+        </SprkListItem>
+        {isHTMLStorybook && (
+          <React.Fragment>
+            <SprkListItem
+              additionalClasses={additionalListItemClasses}
+            >
+              In Vanilla JS applications, if an input is required only,
+              meaning that no validation is needed aside from a value
+              being supplied, you can add the
+              <code> data-sprk-required-only </code>
+              attribute to the input container. The values that are
+              available are &apos;text&apos;, &apos;select&apos;, and
+              &apos;tick&apos;. Text is for any input that accepts text as
+              a value, Select is only for select inputs, and Tick is for
+              radio buttons or checkboxes.
+            </SprkListItem>
+            <SprkListItem
+              additionalClasses={additionalListItemClasses}
+            >
+              You should be providing your own validation for Inputs,
+              not relying on the browser validation. Be sure to add the
+              <code> novalidate </code>
+              attribute to your form element to disable the browser
+              implementation.
+            </SprkListItem>
+            <SprkListItem
+              additionalClasses={additionalListItemClasses}
+            >
+              The Input Label
+              <code> for </code>
+              attribute must match the
+              <code> id </code>
+              attribute of the corresponding Input.
+            </SprkListItem>
+            <SprkListItem
+              additionalClasses={additionalListItemClasses}
+            >
+              The value set in
+              <code> aria-describedby </code>
+              must match the
+              <code> id </code>
+              attribute on the error container.
+            </SprkListItem>
+          </React.Fragment>
+        )}
+      </SprkList>
+    </React.Fragment>
+  );
+};
+
+AdditionalInputInfo.defaultProps = {
+  additionalHeaderClasses: undefined,
```

Comment:
Did the linter complain that you needed default props for these? If not, we don't need to have them.

Reply:
Saw that's how we were doing it in another React component. I removed them!